src/main/java/org/launhcode/healthynutrition/models/Recipe.java

```diff
@@ -52,6 +52,7 @@ public Recipe(String name, String description, int calories, String directions,
     public int getId() {
         return id;
     }
+    public void setId(int id){this.id = id;}
```

Comment:
hmm I don't think you should have this one...

Reply:
Without setting ID, I am not able to get recipe id, when I am editing a recipe

Reply:
Hmm that doesn't make sense to me. You said you need to _get_ the recipe id. I don't see where you are setting it?

Reply:
only the db should set this field